diff -uNr origin/jubatus/anomaly/client.py for_jubatus/jubatus/anomaly/client.py
--- origin/jubatus/anomaly/client.py	2013-03-12 18:33:27.436766999 +0900
+++ for_jubatus/jubatus/anomaly/client.py	2013-03-12 18:34:11.848767001 +0900
@@ -12,6 +12,9 @@
     address = msgpackrpc.Address(host, port)
     self.client = msgpackrpc.Client(address)
 
+  def get_client(self):
+    return self.client
+
   def get_config (self, name):
     retval = self.client.call('get_config', name)
     return retval
diff -uNr origin/jubatus/classifier/client.py for_jubatus/jubatus/classifier/client.py
--- origin/jubatus/classifier/client.py	2013-03-12 18:33:27.444767001 +0900
+++ for_jubatus/jubatus/classifier/client.py	2013-03-12 18:34:11.856767001 +0900
@@ -12,6 +12,9 @@
     address = msgpackrpc.Address(host, port)
     self.client = msgpackrpc.Client(address)
 
+  def get_client(self):
+    return self.client
+
   def get_config (self, name):
     retval = self.client.call('get_config', name)
     return retval
diff -uNr origin/jubatus/graph/client.py for_jubatus/jubatus/graph/client.py
--- origin/jubatus/graph/client.py	2013-03-12 18:33:27.452767001 +0900
+++ for_jubatus/jubatus/graph/client.py	2013-03-12 18:34:11.868767001 +0900
@@ -12,6 +12,9 @@
     address = msgpackrpc.Address(host, port)
     self.client = msgpackrpc.Client(address)
 
+  def get_client(self):
+    return self.client
+
   def get_config (self, name):
     retval = self.client.call('get_config', name)
     return retval
diff -uNr origin/jubatus/recommender/client.py for_jubatus/jubatus/recommender/client.py
--- origin/jubatus/recommender/client.py	2013-03-12 18:33:27.464767001 +0900
+++ for_jubatus/jubatus/recommender/client.py	2013-03-12 18:34:11.876767001 +0900
@@ -12,6 +12,9 @@
     address = msgpackrpc.Address(host, port)
     self.client = msgpackrpc.Client(address)
 
+  def get_client(self):
+    return self.client
+
   def get_config (self, name):
     retval = self.client.call('get_config', name)
     return retval
diff -uNr origin/jubatus/regression/client.py for_jubatus/jubatus/regression/client.py
--- origin/jubatus/regression/client.py	2013-03-12 18:33:27.476767001 +0900
+++ for_jubatus/jubatus/regression/client.py	2013-03-12 18:34:11.884767001 +0900
@@ -12,6 +12,9 @@
     address = msgpackrpc.Address(host, port)
     self.client = msgpackrpc.Client(address)
 
+  def get_client(self):
+    return self.client
+
   def get_config (self, name):
     retval = self.client.call('get_config', name)
     return retval
diff -uNr origin/jubatus/stat/client.py for_jubatus/jubatus/stat/client.py
--- origin/jubatus/stat/client.py	2013-03-12 18:33:27.484767001 +0900
+++ for_jubatus/jubatus/stat/client.py	2013-03-12 18:34:11.892767001 +0900
@@ -12,6 +12,9 @@
     address = msgpackrpc.Address(host, port)
     self.client = msgpackrpc.Client(address)
 
+  def get_client(self):
+    return self.client
+
   def get_config (self, name):
     retval = self.client.call('get_config', name)
     return retval
